lib/Raisin/Encoder/JSON.pm

```diff
@@ -3,12 +3,16 @@ package Raisin::Encoder::JSON;
 use strict;
 use warnings;
 
-use JSON qw(encode_json decode_json);
+use JSON::MaybeXS  qw(encode_json decode_json );
```

Comment:
The code isn't using `encode_json` and `decode_json` functions any longer hence you could safely remove it.

Reply:
done, thx